packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -128,16 +133,42 @@ export const Docs: () => JSX.Element = () => {
       </Description>
 
       <Heading>Adding file sharing</Heading>
+      <Subheading>In Azure Communication Services Chat Thread</Subheading>
       <SingleLineBetaBanner />
       <Description>
-        The CallWithChat Composite supports file sharing capabilities in conjunction with your choice of a storage
-        solution. Using the provided APIs, you can enable the composite to support uploading files and displaying them
-        on the send box before sending, and the message thread once they have been sent or received. For an end to end
-        tutorial on enabling file sharing with Azure Blob Storage as your choice of a storage solution, refer to our
-        [tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial). File
-        sharing is supported for Teams interop scenarios by default.
+        In a chat thread where all participants are Azure Communication Services end users, the `CallWithChat` Composite
+        supports file capabilities in conjunction with your choice of a storage solution. Using the provided APIs, you
+        can composite to support uploading files and displaying them on the send box before sending, and the message
+        once they have been sent or received. For an end to end tutorial on enabling file sharing with Azure Blob
+        Storage as your choice of a storage solution, refer to our
+        [tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial).
       </Description>
       <Source code={addFileSharingSnippet} />
+      <Subheading>In Teams Interop Meeting Chat Thread</Subheading>
+      <SingleLineBetaBanner />
+      <Description>
+        When using the `CallWithChat` Composite in a Teams interop scenario, the Azure Communication Services end user
+        can recieve file attachments from Teams user without any additional setup. Simply join the Teams meeting as an
+        Azure Communication Services end user, as soon as the Teams user sends a file from the Teams client, the Azure
+        Communication Services end user will be able to see shared files in the chat thread.
+      </Description>
+      <Description>
+        Please note there are few things to keep in mind when using file sharing in Teams interop scenario:
+      </Description>
+      <ul style={{ color: 'black' }}>
```

Comment:
Recommend we use a the style in the `preview-head.html` if nothing fits what we are trying to do than lets moves these styles there so it can be shared and reused properly

Reply:
thanks added